crossdock/server/yarpc/sleep.go

```diff
@@ -0,0 +1,12 @@
+package yarpc
+
+import (
+	"time"
+
+	"github.com/yarpc/yarpc-go/encoding/raw"
+)
+
+func SleepRaw(reqMeta *raw.ReqMeta, body []byte) ([]byte, *raw.ResMeta, error) {
+	time.Sleep(1 * time.Second)
```

Comment:
For a separate PR: Can we make this a JSON/Thrift procedure instead? It could accept the amount of time it needs to sleep as an argument.